api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,189 @@
+	</changeSet>
+	<changeSet author="vshankar" id="201609171246-7">
+		<addAutoIncrement tableName="cohort_member" columnName="cohort_member_id" columnDataType="int"/>
+	</changeSet>
+	<changeSet id="201610042145-1" author="vshankar">
+        <preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_created"/>
+			</not>
+		</preConditions>
+        <addColumn tableName="cohort_member">
+			<column name="date_created" type="DATETIME"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-1.1" author="vshankar">
+		<sql>
+			UPDATE cohort_member
+			INNER JOIN cohort USING (cohort_id)
+			SET cohort_member.date_created = cohort.date_created;
+		</sql>
+	</changeSet>
+	<changeSet id="201610042145-1.2" author="vshankar">
+		<addNotNullConstraint tableName="cohort_member" columnName="date_created" columnDataType="DATETIME"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-1">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided"/>
+			</not>
+		</preConditions>
+        <addColumn tableName="cohort_member">
+			<column name="voided" type="BOOLEAN">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-2">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided_by"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="voided_by" type="int"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-3">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_voided"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="date_voided" type="DATETIME"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-4">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="void_reason"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="void_reason" type="varchar(255)"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-2" author="vshankar">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="uuid"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="uuid" type="char(38)"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-2.1" author="vshankar">
+		<preConditions onFail="MARK_RAN"><or><dbms type="mysql"/><dbms type="oracle" /></or></preConditions>
+		<comment>Generating UUIDs for all rows in cohort_member table via built in uuid function.</comment>
+		<update tableName="cohort_member" >
+			<column name="uuid" valueNumeric="name-of-uuid-function" />
+			<where>uuid is null</where>
+		</update>
+		<modifySql dbms="mysql"><replace replace="name-of-uuid-function" with="UUID()"/></modifySql>
```

Comment:
@rkorytkowski , @dkayiwa do we still need to specify the DB like here? I recall as though we decided that we assume mysql for upgrade scripts